app/views/users/sessions/new.html.erb

```diff
@@ -1,50 +1,50 @@
 <div class="container">
   <div class="row">
-    <div class="col-xs-12 col-sm-12 col-md-6 col-lg-6 center-image">
+    <div class="col-12 col-sm-12 col-md-6 col-lg-6 center-image">
       <%= image_tag "SVGs/login.svg", alt: "Login Sketch", class: "users-image" %>
       <h5><%= t("welcome_back_text") %></h5>
     </div>
 
-    <div class="col-xs-12 col-sm-12 col-md-6 col-lg-6">
+    <div class="col-12 col-sm-12 col-md-6 col-lg-6">
       <div class="form-container">
         <h1 class="users-main-heading"><%= t("login") %></h1>
         <%= form_with(model: resource, as: resource_name, url: session_path(resource_name), local: true) do |f| %>
         <% flash.each do |key, value| %>
           <div class="error-message"><%= value %></div>
         <% end %>
-        <div class="field form-group">
+        <div class="field mb-3">
           <h6><%= f.label :email %></h6>
           <div class="input-group">
             <%= f.email_field :email, autofocus: true, placeholder: t('enter_email'),class: "form-control form-input user-email-input" %>
           </div>
         </div>
-        <div class="field form-group">
+        <div class="field mb-3">
           <h6><%= f.label :password %></h6>
           <div class="input-group form-input">
             <%= f.password_field :password, autocomplete: "off", placeholder:t('enter_password') , class: "form-control users-password-input" %>
-            <div class="input-group-addon ml-3 mt-2" onclick="togglePassword()">
+            <div class="input-group-addon ms-3 mt-2" onclick="togglePassword()">
               <i class="far fa-eye-slash password-icon" aria-hidden="true" id="toggle-icon"></i>
             </div>
           </div>
```

Comment:
the current inline onclick implementation is straightforward and works reliably within our setup. Refactoring this would add complexity and might introduce unexpected behavior if not fully tested across different browsers and devices. For now, I'd prefer to keep this as-is to avoid potential compatibility issues and keep the code simple.